conformance/conformance.go

```diff
@@ -209,19 +203,28 @@ func RunConformanceWithOptions(t *testing.T, opts confsuite.ConformanceOptions)
 	cSuite, err := confsuite.NewConformanceTestSuite(opts)
 	require.NoError(t, err, "error initializing conformance suite")
 
+	installedCRDs := &apiextensionsv1.CustomResourceDefinitionList{}
+	err = opts.Client.List(context.TODO(), installedCRDs)
```

Comment:
I noticed the use of context in conformance is not aligned with best practices.
I was hoping to see context created one time in "main" (e.g., in the entry point), and then ctx is propagated in the function calls (usually as first arg).
I've never ran the conformance test suite myself, but if it's a long running process (more than few minutes), it may make sense to use context with signal handling (listening to SIGTERM, SIGKILL). 
see example in our main file where we use `SetupSignalHandler()`:
https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/6491db2dc78db2217ffac9d98964235e00d98ca5/cmd/epp/main.go#L31